engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-subscriptions-modal.controller.js

```diff
@@ -30,6 +30,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkSubscription
             function getBulkSubscriptionParams() {
                 var bulkParams = hostIds;
                 bulkParams.subscriptions = SubscriptionsHelper.getSelectedSubscriptionAmounts($scope.table);
+                bulkParams.organization_id = CurrentOrganization; // eslint-disable-line camelcase
```

Comment:
Maybe we can just do `bulkParams['organization_id']` to work around the lint?

Reply:
:exploding_head: